server/service/async/async.go

```diff
@@ -3,25 +3,16 @@ package async
 import (
 	"context"
 	"fmt"
-	"math"
```

Comment:
Most of the code churn in this file is due to reorganization - `async.go` contains code common to all async processing, label-specific code is now in `async_label.go`, and the new policy-specific code is in `async_policy.go`.